setup.py

```diff
@@ -14,7 +14,7 @@
     python_requires='>=3.5',
     install_requires=[
         'Django>=1.11',
-        'asgiref~=3.0',
+        'asgiref~=3.2',
```

Comment:
Do we **need** to bump this? `~=3.0` is the same as `>=3.0` no, so it'll pick up the latest version?